### DISCUSSION TITLE
Welcome to jvcl Discussions!

### DISCUSSION THREAD

**Discussion by obones** [project-jedi/jvcl#163] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ronaldhoek** [comment]
I've been using Delphi for several years (25) and have created plenty of useful applications (mostly internal tools). But some are still live customer applications running in high-volume environments. 
In those years I started to use the JEDI libraries (JCL/JVCL) - at first as RxLib and later as the project JEDI library - and also contributed some parts to the code (new features en bug fixes) where possible.

However, after debating for a long time, we as a company moved to another environment (.NEt / Angular) to develop our new browser-based application.

But since my heart still lies with Delphi (and we still have some live Delphi applications), I will keep an eye on this community and keep contributing when possible!

**Comment by jfudickar** [comment]
Hi,
i started my development career in 1987 with Turbo Pascal 3. Since then I've used all turbo pascal releases and Delphi from version 1:-)
I started with JCL/JVCL some years ago and put a lot off effort in the AppStore components, some database stuff and the DynControl components.
I'm developing for 20 years now an Oracle Support Tool, but now only in my free time.
And the free time is getting less and less and in my compnay I'm no longer working with oracle, so the need to enhance the tool is getting less and less.
And so now I'm mainly watching and fixing things I easily can fix.
Jens

**Comment by maxkleiner** [comment]
Me too I've been using Delphi since the beginning at 1995 and created 
plenty of stunning applications like maXbox scripting engine and maXbox 
studio. But some are still live industrial applications running in 
logistic and security environments demonstrated at all EKONs.
Max

---
https://maxbox4.wordpress.com/

*******************************************

On 2022-09-30 10:15, obones wrote:

> 👋 WELCOME!
> 
> We're using Discussions as a place to connect with other members of our 
> community. We hope that you:
> 
> * Ask questions you're wondering about.
> * Share ideas.
> * Engage with other community members.
> 
> * Welcome others and are open-minded. Remember that this is a community 
> we
> build together 💪.
> 
> To get started, comment below with an introduction of yourself and tell 
> us about what you do with this community.
> 
> --
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [2].
> You are receiving this because you are subscribed to this 
> thread.Message ID: ***@***.***>


Links:
------
[1] https://github.com/project-jedi/jvcl/discussions/163
[2]

**Comment by maurizuki** [comment]
Hi,

I started my working experience with Delphi 3 in 1999 then I changed to .NET / Javascript frameworks after fifteen years.
I published the first version of my open source app [O2](https://github.com/maurizuki/O2) in 2004: it's a safe storage for personal data built with Dephi using JCL / JVCL. I continue to develop it in free time, if you are interested to contribute, check the [open issues](https://github.com/maurizuki/O2/issues)  and reply to the [welcome discussion](https://github.com/maurizuki/O2/discussions/14).

Maurizio 🚀
